api/app/forecasts/noon_forecasts.py

```diff
@@ -19,9 +18,9 @@ class StationNotFoundException(Exception):
     """ Custom exception for when a station cannot be found """
 
 
-def parse_table_records_to_noon_forecast_response(data: [NoonForecasts]):
+def parse_table_records_to_noon_forecast_response(data: [app.db.models.NoonForecast]):
```

Comment:
does this work even though you're not importing app.db.models?

Reply:
It shouldn't, right?
I am guessing it works because db is imported through `__init__.py`?

Reply:
Weird.

Reply:
I will add that anyway just in case!